rrpproxy/rrpproxy.py

```diff
@@ -8,7 +8,11 @@
 
 
 class RRPProxy:
-    def __init__(self, username, password, use_test_environment=False):
+    def __init__(self, username, password, use_test_environment=False, session=None):
```

Comment:
Nitpick: It might be more logical to have `session` before `use_test_environment`. I think it the `session` parameter would be used more often. If you are using args instead of kwargs that would look a little cleaner.

Reply:
I'll do this now, but normally this would cause it to not be backwards compatible anymore.